pkg/ring/lifecycler_test.go

```diff
@@ -26,7 +29,7 @@ func (f *flushTransferer) TransferOut(ctx context.Context) error {
 	return f.lifecycler.ChangeState(ctx, ACTIVE)
 }
 
-func testLifecyclerConfig(ringConfig Config, id string) LifecyclerConfig {
+func testLifecyclerConfig(t *testing.T, ringConfig Config, id string) LifecyclerConfig {
```

Comment:
You don't need to pass `t` here. Can you rollback it, please?

Reply:
Another cleanup left from the previous :/